turbinia/evidence.py

```diff
@@ -536,16 +540,18 @@ class RawDisk(Evidence):
     device_path (str): Path to a relevant 'raw' data source (ie: a block
         device or a raw disk image).
     mount_partition: The mount partition for this disk (if any).
-    size: The size of the disk in bytes.
   """
 
   POSSIBLE_STATES = [EvidenceState.ATTACHED]
 
-  def __init__(self, size=None, *args, **kwargs):
+  def __init__(self, *args, **kwargs):
     """Initialization for raw disk evidence object."""
     self.device_path = None
-    self.size = size
     super(RawDisk, self).__init__(*args, **kwargs)
+    if hasattr(config, 'TURBINIA_COMMAND') and config.TURBINIA_COMMAND:
```

Comment:
Maybe instead of `if 'worker` in config.TURBINIA_COMMAND can just do 
```if hasattr(config, 'TURBINIA_COMMAND') and 'worker' in config.TURBINIA_COMMAND```

Reply:
Also what's the need for `TURBINIA_COMMAND`?  Is it because to prevent server from running via `__init__`. Would it be better to put this under `_preprocess` code which is where mounting happens/don't need to do check there?

Reply:
I initially thought it might be better to populate the size on initialisation, but I think pre-processing works.

Reply:
Perfect sounds good,  we also have a locking mechanism in `preprocess` and `postprocess` code to allow for multiple workers on same host to process same evidence so was also considering that (although this Evidence type might not be affected but will prevent issues down the line)